src/core/regimes.rkt

```diff
@@ -208,7 +211,7 @@
                      [*var-reprs* (dict-set (*var-reprs*) var repr)])
         (define ctx
           (prepare-points start-prog
-                          `(λ ,(program-variables start-prog) (== ,(caadr start-prog) ,v))
+                          `(λ ,(program-variables start-prog) (,eq-repr ,(caadr start-prog) ,v))
```

Comment:
Another great find. Long term, the `get-parametric-operator` API should be cleaner (added a Trello card).